industrial_ci/src/tests/source_tests.sh

```diff
@@ -173,10 +173,12 @@ function run_source_tests {
     ici_step "setup_rosdep" ici_setup_rosdep
 
     extend=${UNDERLAY:?}
+    export ROSDEP_SOURCE_FOLDERS=("${UNDERLAY:?}")  # source folders to be ignored for rosdep install
```

Comment:
I think this should go into env.sh

Reply:
This variable is configured only here (incrementally for all stages).
Moving the initialization into another file seems odd.